AdyenNetworking/Helpers/Logging.swift

```diff
@@ -6,59 +6,59 @@
 
 import func Darwin.fputs
 import Foundation
+import OSLog
 
 /// Provides control over SDK logging.
 public enum Logging {
     /// Indicates whether to enable printing to the console.
     public static var isEnabled: Bool = false
 }
 
-@_spi(AdyenInternal)
-public func adyenPrint(_ items: Any..., separator: String = " ", terminator: String = "\n", fileId: String = #fileID) {
-    guard Logging.isEnabled else { return }
-    let moduleName = fileId.split(separator: "/").first ?? "AdyenNetworking"
+extension Logging {
     
-    var items = items
-    items.insert("[\(moduleName)]", at: 0)
-    items.insert(ISO8601DateFormatter().string(from: Date()), at: 0)
+    @available(iOS 14.0, *)
+    static var logger: Logger { return Logger(subsystem: Bundle.main.bundleIdentifier ?? "com.adyen.Adyen", category: "AdyenNetworking") }
```

Comment:
do you want to use the merchant app `bundleIdentifier` or the `AdyenNetworking` bundle identifier? 🤔 for better categorization of logs maybe we should use the `AdyenNetworking` bundle identifier instead?

Reply:
Yes! Good point, will adjust after trying it out with Xcode 15 👍